extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -63,6 +63,7 @@ require([
 			return {
 				author: thread.createdBy.name,
 				authorAvatar: thread.createdBy.avatarUrl,
+				authorUserPageUrl: window.wgScriptPath + '/wiki/User:' + thread.createdBy.name,
```

Comment:
Maybe
```javascript
authorUserPageUrl: mw.util.getUrl('User:' + thread.createdBy.name),
```
? (Using `wg` variables from global scope is deprecated on newer MediaWiki versions.)

Reply:
yup, I know, however if we'll do similar module in the UCP platform we'll probably write it from scratch instead of porting this code. For this legacy platform I'd not worry much about stuff like that